packages/router/dist/index.d.ts

```diff
@@ -0,0 +1,4 @@
+import EmberRouter from '@ember/routing/router';
```

Comment:
should we gitignore `packages/router/dist/` folder?

Reply:
good catch. I used the v2 addon blueprint but then lifted the actual addon folder out of the generated monorepo, and that left the default gitignore behind.